### ISSUE TITLE
Add skip option

### ISSUE THREAD

**Issue by janArb** [slem1/await-maven-plugin#4]
Hey,

thanks for the plugin, we use it for our integration tests. Could you extend the plugin to have a skip option like most plugins in case of running without tests for example:

```
<configuration>
    <skip>${maven.test.skip}</skip>
</configuration>
```

Best regards
Jan

**Comment by slem1** [comment]
Hi, good idea, I created a PR, tell me if it's match what you need. 

I will give it some test run by the end of the day,

**Comment by slem1** [comment]
Hi @janArb , good idea, I created a PR, tell me if it's match what you need.

I will give it some test run by the end of the day,

**Comment by slem1** [comment]
Fixed in 1.2.1 version

**Comment by janArb** [comment]
Hey @slem1, I needed a moment to realize you called the parameter awaitSkip instead of skip, but you put that in the documentation, so I found my first issue. Works as expected, thanks.

I would still prefer to have the wording equal to surefire plugin, exec maven plugin, maven compiler plugin, don't know which other plugins, they all call the parameter only "skip". My open issue is a very basic maven one: I personally use `-DskipTests` instead of `-Dmaven.test.skip` to have at least feedback compilation has worked, but this plugin skip logic is really limited of having only one property, you cannot combine these two properties with boolean OR logic and would need profiles for this again. Thanks for your update nevertheless!

PS: Other plugins display a hint for the skip, that can help to avoid confusion:
```
[INFO] --- await-maven-plugin:1.2.1:Await (default) @ Suite ---
[INFO]
[INFO] --- maven-surefire-plugin:2.18.1:test (default-test) @ Suite ---
[INFO] Tests are skipped.
```
